acceptance-tests/simple-permissioning-smart-contract/migrations/2_deploy_contracts.js

```diff
@@ -1,5 +1,7 @@
 var SimplePermissioning = artifacts.require("SimplePermissioning");
+var SimpleAccountPermissioning = artifacts.require("SimpleAccountPermissioning");
 
 module.exports = function(deployer) {
   deployer.deploy(SimplePermissioning);
```

Comment:
Should we rename `SimplePermissioning` to SimpleNodePermissioning`?

Reply:
done